### PR TITLE
feat: added files for card component

### PR DESCRIPTION
**Pull request by Ranjeetk489**

### CODE REVIEW

File: components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  height: 12.5rem;
+}
+
+.ecom-card {
+  padding: 1.4em;
+}
+.ecom-card:hover {
+  box-shadow: rgba(0, 0, 0, 0.56) 0px 22px 70px 4px;
+}
+.ecom-img {
```
Comment (sanjitsarkar) [inline components/Card/card.css:99]: make it universal for all purpose.
Don't put styling for specific to one category like e-commerce

**Review by sanjitsarkar — APPROVED** [review]

File: components/Card/card.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+.card-hz-header {
+  padding: 2em;
+  background-color: var(--warning-color);
+  height: 12.5em;
+}
+.card-subtitle {
+  padding: 8px;
+}
+.card-controls {
+  /* margin-left: 1em; */
```
Comment (DivyanshSareen) [inline components/Card/card.css:77]: remove comment

**Review by DivyanshSareen — APPROVED** [review]
good to merge

File: components/Card/card.css (hunk 1 above)
Comment (Ranjeetk489) [reply components/Card/card.css:99]: Okay I'll edit the code .. thanks for reviewing

**Pull request merged**